### PR TITLE
Force apt to install the things we asked it to

### PR DESCRIPTION
**Pull request by aqk**

### CODE REVIEW

File: .github/workflows/benchmarks.yml [hunk 1]
```diff
@@ -51,11 +51,13 @@ jobs:
           ${{ runner.os }}-pip-
 
     - name: Install ubuntu dependencies
+      env:
+        DEBIAN_FRONTEND=noninteractive
```
Comment (xdustinface) [inline .github/workflows/benchmarks.yml:55]:
```suggestion
        DEBIAN_FRONTEND: noninteractive
```

Not sure why this is needed? But i think this should be like ^

Comment (xdustinface) [reply .github/workflows/benchmarks.yml:55]: Oh, just saw the chat in IPS channel. Sounds like we could just drop it anyway?

Comment (aqk) [reply .github/workflows/benchmarks.yml:55]: Thanks - is this how we set env variables?

Comment (xdustinface) [reply .github/workflows/benchmarks.yml]: Yeah, the Benchmark workflow didn't even run before. Obviously because of wrong syntax.

**Review by xdustinface — APPROVED** [review]
Seems to work now and fixes the issue 👍

**Comment by aqk** [comment]
Thanks!

**Review by wjblanke — APPROVED** [review]
aok

**Pull request merged**